src/inference/include/ie/gpu/gpu_config.hpp

```diff
@@ -130,6 +130,14 @@ INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_GPU_CONFIG_KEY(MAX_NUM_THREADS);
  * Thus, this key should be turned off if graph loading time is considered to be most important target to optimize.*/
 INFERENCE_ENGINE_1_0_DEPRECATED DECLARE_GPU_CONFIG_KEY(ENABLE_LOOP_UNROLLING);
 
+/**
+ * @brief Turning on this key disable winograd convoltuion in iGPU.
+ * Winograd's algorithm for convolutions can greatly reduce the number of arithmetic operations.
+ * However a large reduction in floating point (FP) operations in these algorithmscan result in reduced FP accuracy of
+ * the result. Default value is false.
```

Comment:
I think these three lines are too much detail. We can simplify mention like this: `Winograd convolution has different characteristics for accuracy and performance compared to other convolution implementations.`

Reply:
Applied